src/components/footer.vue

```diff
@@ -62,17 +96,47 @@
 </style>
 
 <script>
+import axios from "axios";
 export default {
   data: () => ({
     email: "",
-    emailIsValid: false
+    snackbar: false,
+    message: "",
+    emailIsValid: false,
   }),
   methods: {
+    async submitEmail() {
+      await axios
+        .get(
+          "https://www.nexd.app/.netlify/functions/mailchimp?email=" +
```

Comment:
```suggestion
          "/.netlify/functions/mailchimp?email=" +
```
You can simply use this on the current host, which will allow to work then also for previews and other domains.